source/_patterns/01-atoms/badge/demo/badges.yml

```diff
@@ -0,0 +1,109 @@
+badge_helper:
+  card_title: Badges
+  card_text: "
+             <p><code>badge_element</code> String: span|a choose the element type for your badge. If you use an link element make sure to enter a value for badge_link</p>
+             <p><code>badge_pill</code> Boolean: TRUE|FALSE make badges more rounded</p>
+             <p><code>badge_color</code> String: primary|secondary|success|danger|warning|info|light|dark the color of the badge</p>
+             <p><code>badge_link</code> String: the URL the badge will lead to. Make sure to set badge_element to 'a'</p>
+             <p><code>badge_text</code> String: display text on the badge</p>"
+  text_color: light
+  card_background: secondary
+
+sizes:
+  - badge_color: primary
+    badge_text: Badge
+    badge_size: h1
+  - badge_color: primary
+    badge_text: Badge
+    badge_size: h2
+  - badge_color: primary
+    badge_text: Badge
+    badge_size: h3
+  - badge_color: primary
+    badge_text: Badge
+    badge_size: h4
+  - badge_color: primary
+    badge_text: Badge
+    badge_size: h5
+  - badge_color: primary
+    badge_text: Badge
+    badge_size: h6
+
+button_badge:
+  button_color: primary
+  button_text: Notifications
+colors:
```

Comment:
What if we just had a universal set of values on `_data.json`? ie:

```json
{
  "colors": [ "primary", "secondary", "etc"]
}
```

Do the same thing for sizes, etc from Bootstrap. Looping through the same "things" in every demo doesn't speak to the "systems" at play across the board.